config/deployments/control-plane/kustomization.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+namespace: k8ssandra-operator
+
 resources:
-  - ../default
-  - github.com/k8ssandra/cass-operator/config/deployments/default?ref=v1.9.0
+  - ../../default
```

Comment:
We should keep this pointing at `config/deployments/default` because at release time that is where the image tag is updated, not in `config/default`.

Reply:
Sure, I can see that would be a problem. Will change it back.

It'd be nice to remove `config/deployments/default` at some stage to simplify things, since it actually replicates exactly what's under `config/default` (which e.g. also has an image override).